src/app/talent/talent/talent.component.ts

```diff
@@ -126,10 +126,52 @@ export class TalentComponent extends AbstractComponent<Talent> {
 		return mediasToUpdate;
 	}
 
+	public doSomething() {
+		console.log("merge");
+	}
+
+
 	public update() {
-		super.update();
-		const mediasToUpdate = this.getModifiedMedias();
-		this.mediaService.updateMedias(mediasToUpdate).subscribe((data) => {});
+		// super.update();
+		// const mediasToUpdate = this.getModifiedMedias();
+		// this.mediaService.updateMedias(mediasToUpdate).subscribe((data) => {});
+
+		const profilePromise = this.getImageFile(this.profileCroppedImage, 'profileCropped');
+		const listingPromise = this.getImageFile(this.listingCroppedImage, 'listingCropped');
+
+		Promise.all([profilePromise, listingPromise]).then((values) => {
+			const profile: FormFile = {
+				file: this.profileFile,
+				key: 'profileImage'
+			};
+
+			const profileCropped: FormFile = {
+				file: values[0],
+				key: 'profileCroppedImage'
+			};
+
+			const listing: FormFile = {
+				file: this.listingFile,
+				key: 'listingImage'
+			};
 
+			const listingCropped: FormFile = {
+				file: values[1],
+				key: 'listingCroppedImage'
+			};
+			this.item['medias'] = this.getModifiedMedias();
+			this.service.updateForm(this.item, [profile, profileCropped, listing, listingCropped]).subscribe((data) => {
+				console.log(data);
+			});
+
+		});
+	}
+
+	private checkButtonStatus() {
```

Comment:
you can do "return !(this.listingImage && this.profileImage)" which returns a boolean, true if it would enter in the if statement, false if it wouldn't, so exactly what you need